test/cmocka/src/audio/eq_iir/CMakeLists.txt

```diff
@@ -13,7 +13,7 @@ add_compile_options(-DUNIT_TEST)
 
 add_library(audio_for_eq_iir STATIC
 	${PROJECT_SOURCE_DIR}/src/audio/eq_iir/eq_iir.c
-	${PROJECT_SOURCE_DIR}/src/audio/eq_iir/iir.c
+	${PROJECT_SOURCE_DIR}/src/math/iir.c
```

Comment:
once I get around to removing dll stuff in the testbench changes like this should no longer be needed.... but my list of things to do before that is quite long :upside_down_face: